docs/release_process.md

```diff
@@ -140,6 +144,18 @@ After the release branches are created, preliminary testing should be conducted
         - [ ] unit tests pass
         - [ ] program starts up correctly
         - [ ] GUI functions & can display messages
+    - [ ] jpo-utils
+        - [ ] jpo-deduplicator
+            - [ ] deduplicator code compiles
+            - [ ] deduplicator unit tests pass
+            - [ ] program starts up correctly
+            - [ ] program processes BSM, SPaT, MAP, and TIM messages
+            - [ ] ProcessedMap, ProcessedSpat, OdeMap, OdeTim, OdeRawEncodedTim, OdeBsm messages are consumed
+            - [ ] A single message is output for each of the input types
+        - [ ] jpo-jikkou docker image builds
+        - [ ] jpo-kafka-connect image builds
+            - [ ] Verify all connectors are created using connect rest api
```

Comment:
When do we want to update the submodule references to the `jpo-utils`  repository (in the ODE, geojsonconverter, etc.)?